src/torrent/info.rs

```diff
@@ -107,10 +109,12 @@ impl Info {
                 })
             })
             .ok_or("No hash found in magnet")?;
-        let announce = url.query_pairs()
-            .find(|&(ref k, _)| k == "tr")
-            .map(|(_, ref v)| v.to_string())
-            .unwrap_or_else(|| "".to_owned());
+        let announce = url.query_pairs().find(|&(ref k, _)| k == "tr").and_then(
+            |(_,
+              ref v)| {
```

Comment:
This and the previous line should be combined.

Reply:
The version of rustfmt on travis doesn't like that. https://travis-ci.org/Luminarys/synapse/jobs/317495779#L491